Add system idea (#5)
appuio/appuio-io-docs: docs/modules/ROOT/pages/explanation/system/idea.adoc

= System idea
:apub2: APPUiO Public 2.0

The system idea displays a high-level architecture of {apub2}.

.{apub2} system idea
image::system/systemidea.drawio.svg[]

* An instance of Keycloak is responsible for authoritatively store data on:
** Organizations
** Teams
** Users

> * Data owned by Keycloak will be cached on each Zone.

Reviewer: ```suggestion
* Data owned by Keycloak will be replicated to each Zone.
```
